Formula/ooniprobe.rb

```diff
@@ -15,275 +15,108 @@ class Ooniprobe < Formula
-
-    virtualenv_install_with_resources
+    (buildpath/"src/github.com/ooni/probe-cli").install buildpath.children
 
-    man1.install Dir["data/*.1"]
-    pkgshare.install Dir["data/*"]
+    cd "src/github.com/ooni/probe-cli" do
+      system "./build.sh macos"
+      bin.install "CLI/darwin/amd64/ooniprobe"
+    end
+    ooni_home = Pathname.new "#{var}/ooniprobe"
+    ooni_home.mkpath
   end
 
   def post_install
-    return if (pkgshare/"decks-available").exist?
-
-    ln_s pkgshare/"decks", pkgshare/"decks-available"
-    ln_s pkgshare/"decks/web.yaml", pkgshare/"current.deck"
+    ENV["OONI_HOME"] = "#{HOMEBREW_PREFIX}/var/ooniprobe"
+    system bin/"ooniprobe", "onboard", "--yes"
   end
 
   def caveats
     <<~EOS
-      Decks are installed to #{opt_pkgshare}/decks.
+      By enabling the homebrew service you will not be shown the informed consent.
+
+      WARNING:
+
+      • OONI Probe will likely test objectionable sites and services
+      • Anyone monitoring your internet activity (such as your government
+        or Internet provider) may be able to tell that you are using OONI Probe
+      • The network data you collect will be published automatically
+
+      To learn more about the risks see:
+          https://ooni.org/about/risks
     EOS
   end
 
-  plist_options startup: "true", manual: "ooniprobe -i #{HOMEBREW_PREFIX}/share/ooniprobe/current.deck"
+  plist_options manual: "ooniprobe run"
 
   def plist
     <<~EOS
       <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
       <plist version="1.0">
       <dict>
-        <key>Label</key>
+          <key>Label</key>
           <string>#{plist_name}</string>
-        <key>EnvironmentVariables</key>
-        <dict>
-          <key>PATH</key>
-          <string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
-        </dict>
-        <key>ProgramArguments</key>
-        <array>
-          <string>#{opt_bin}/ooniprobe-agent</string>
-          <string>run</string>
-        </array>
-        <key>RunAtLoad</key>
-          <true/>
-        <key>KeepAlive</key>
+
+          <key>KeepAlive</key>
+          <false/>
+          <key>RunAtLoad</key>
           <true/>
-        <key>StandardErrorPath</key>
-          <string>/dev/null</string>
-        <key>StandardOutPath</key>
-          <string>/dev/null</string>
-        <key>WorkingDirectory</key>
-          <string>#{opt_prefix}</string>
+
+          <key>EnvironmentVariables</key>
+          <dict>
+            <key>OONI_HOME</key>
+            <string>#{HOMEBREW_PREFIX}/var/ooniprobe</string>
+          </dict>
+
+          <key>ProgramArguments</key>
+          <array>
+              <string>#{opt_bin}/ooniprobe</string>
+              <string>--log-handler=syslog</string>
```

Comment:
Why specifically syslog?

Reply:
Because syslog directs the logs to Apple's logger. So a user can open /Applications/Utility/Console.app and filter for `ooniprobe` to observe logs generated by us. But we don't write files on disk and we comply as much as possible to what seem Apple expectations from developers (AFAICT, but correct me if wrong). The specific implementation is using libc's `syslog(3)`, which seem to do the right thing for our user case (i.e., logs safely land in Console.app).

Reply:
Is this the way a normal user would run `ooniprobe`? I personally wouldn't expect non-system programs to show up in the Console app.

Reply:
So that it ends up inside of the Console app in macOS and doesn't leave files on disk, potentially occupying a lot of on-disk storage.

Reply:
A normal user runs `ooniprobe run [...]` from the console and gets messages on the console. This part of the diff is creating a `.plist` such that a user could configure `brew services start ooniprobe` as a launch agent that is periodically run in the background in the context of the user session. The choice to log on the systlog, therefore, only pertains to the use case where we're running in the background.

Reply:
I understand that this is a service, but that should be running as the default state that upstream defined regardless of your views on disk usage by homebrew formulae. If upstream says "make a systemd unit with `/usr/bin/x -a -b`" then the plist file should also be made that way.

Reply:
We are the upstream (both @hellais and me are core developers of @ooni). This is the recommended way in which we want people to run ooniprobe when running in the background :-).